public/app/plugins/datasource/cloudwatch/dashboards/Lambda.json

```diff
@@ -187,27 +457,28 @@
       "steppedLine": false,
       "targets": [
         {
+          "alias": "",
```

Comment:
By using `{{label}}`, the legend becomes FunctionName.

Reply:
That is actually already the case when using wildcards.